src/BPSecLib_Public.h

```diff
@@ -186,6 +186,18 @@ typedef struct BSL_BundleRef_s
     void *data; ///< Opaque pointer, not used by the BSL.
 } BSL_BundleRef_t;
 
+typedef enum
```

Comment:
Please cite the IANA registry explicitly, either link or with appropriate `@cite`. It would be helpful, but not mandatory to copy in the IANA "description" as an item `///` comment to make absolutely clear the interpretation of these names (especially because we are not defining these codes or descriptions).